.env.example

```diff
@@ -1,2 +1,10 @@
 MAPBOX_ACCESS_TOKEN=''
 SERVER_URL=''
+AUTH_URL=''
+AUTH_AUTHORITY=''
+AUTH_ID=''
+AUTH_TYPE=''
+AUTH_SCOPE=''
+AUTH_LOGIN_URL=''
```

Comment:
For later:
Remove some unnecessary vars from this .env file.

Reply:
Done

Reply:
Done.